test/e2e/resource_quota_error_test.go

```diff
@@ -40,10 +40,11 @@ func TestResourceQuotaError(t *testing.T) {
 
 	clients := test.Setup(t, test.Options{Namespace: "rq-test"})
 	const (
-		errorReason    = "RevisionFailed"
-		errorMsgScale  = "Initial scale was never achieved"
-		errorMsgQuota  = "forbidden: exceeded quota"
-		revisionReason = "ProgressDeadlineExceeded"
+		errorReason       = "RevisionFailed"
+		containerCreating = "ContainerCreating"
```

Comment:
Curious - how do you get into this scenario?

Reply:
From a UX perspective I don't know if users will understand what `ContainerCreating` means as a reason - what's the exact message when this happens?

It seems like `RevisionFailed` should always remain the `Reason` at this level of abstraction

Reply:
The message when container is creating is empty, is not a failure is just a "not ready"

The thing is with this test we are bound to what happens first. If the PA propagate its status first then the ContainerCreating will be the first message then the error and so on. Let me try to refactor this test today and see how I can improve it.

Reply:
Done with the refactor! This is ready for another review =)